internal/plugintest/working_dir.go

```diff
@@ -444,3 +526,38 @@ func (wd *WorkingDir) Schemas(ctx context.Context) (*tfjson.ProviderSchemas, err
 
 	return providerSchemas, err
 }
+
+func (wd *WorkingDir) Query(ctx context.Context) ([]string, error) {
```

Comment:
This function is a temporary solution until https://github.com/hashicorp/terraform-exec supports the `query` subcommand.

Reply:
https://github.com/hashicorp/terraform-exec/pull/525 !

Reply:
I've merged the PR:
```
go get github.com/hashicorp/terraform-exec@061a850a52d2e3a408d33066da3601983067b3d9
```